js_modules/dagit/packages/core/src/runs/LogsToolbar.tsx

```diff
@@ -171,6 +172,9 @@ const ComputeLogToolbar = ({
         <Select
           disabled={!steps.length}
           items={Object.keys(logCaptureSteps)}
+          itemPredicate={(query, item) =>
+            item.toLocaleLowerCase().includes(query.toLocaleLowerCase())
```

Comment:
Should we use a Fuze?

Reply:
I think let's keep it simple for now, and maybe come back to it if it looks like fuzzy search would be a big win here.